.dockerignore

```diff
@@ -13,8 +13,6 @@ docs/
 tools/
 
 Cargo.lock
-load_test.sh
```

Comment:
was this already removed?

Reply:
It was not removed in this PR, but I do not see it in the repository. Since I was removing the other one from this file, I figured I'd clean this up as well.